target/linux/realtek/files-5.10/arch/mips/rtl838x/setup.c

```diff
@@ -28,6 +29,46 @@
 
 extern struct rtl83xx_soc_info soc_info;
 
+void plat_get_frequencies(u32 *cpu, u32 *lxb, u32 *mem)
+{
+	struct device_node *np;
+	struct clk *clk;
+
+	*cpu = 500000000; // fallback
+	*lxb = 0;
+	*mem = 0;
+
+	clk = clk_get(NULL, "cpu_clk");
+
+	if (IS_ERR(clk)) {
+		pr_err("no clock driver, using DT.\n");
+		np = of_find_node_by_name(NULL, "cpus");
+		if (!np) {
+			pr_err("No 'cpus' DT node, using default.");
```

Comment:
```suggestion
			pr_err("No 'cpus' DT node, using default.\n");
```
Using default what, default CPU or default CPU freq, if the later; maybe print it to inform the user?
```suggestion
			pr_err("No 'cpus' DT node, using default: '%ld'\n", cpu);
```